tasks/release.js

```diff
@@ -9,7 +9,7 @@ function setUpTaskRelease(tyRun) {
     checks: [],
     options: [],
     onInit: function onInitReleaseTask() {
-      var gulp        = require('gulp'),
+      var gulp        = options.gulp || require('gulp'),
```

Comment:
Surely we can guarantee gulp in the context. If not we need to fail. To load gulp may have unexpected consequences.